views/users/new.pug

```diff
@@ -0,0 +1,14 @@
+extends ../application.pug
+
+block content
+  .row
+  .col-xs-6.offset-xs-3
+    h1 Register a user
+    hr
+    form(action='/users' method="POST")
+      .form-group
+        input.form-control(type='email' placeholder="jekkemail" name='email' required='')
```

Comment:
use " for your attributes also required attribute doesn't need a value thus just required is enough.